protocols/migration/migration_reports_6_0_0_to_reports_5_0_0.py

```diff
@@ -37,7 +197,7 @@ def migrate_interpretation_request_cancer(self, old_instance):
         if new_instance.cancerParticipant and new_instance.cancerParticipant.tumourSamples:
             samples = new_instance.cancerParticipant.tumourSamples
             for sample in samples:
-                if sample.diseaseType in (diseaseType.ENDOCRINE, diseaseType.OTHER):
+                if sample.diseaseType == diseaseType.ENDOCRINE:
```

Comment:
Is this right?

Reply:
that doesn't look right to me, I'll change it

Reply:
changed back in https://github.com/genomicsengland/GelReportModels/pull/324/commits/b6408f5cb2cf21ff2b07f973d913f69ef4b13906